### ISSUE TITLE
Can't remove theme

### ISSUE THREAD

**Issue by phenrypereira** [vinceliuice/WhiteSur-kde#81]
I installed the theme from the Kvantum Manager and when I uninstalled it, I also did the process through the application, which has no effect. In the KDE theme settings, WhiteSur not only continues to appear, but can be applied as if nothing has been done.

I also tried removing the theme using uninstall.sh, which in the terminal outputs "Successful", but again nothing changes.

Finally, hunting in the .themes home folder and .config directories I found some traces of the theme and removed them manually: no effect. In the system directories from /usr/share in the folders of icons, wallpapers, kvantum, colors I removed everything I found with the name WhiteSur and there are still global themes, application styles, window decorations, presentation screens and SDDM (the only one I was able to remove through the KDE settings application, without error messages).

In summary, I want to know what other directories are missing. I don't want any sign of the theme in the system.

Thank you for your willingness to solve my problem
